rfc/draft-oberstet-hybi-tavendo-wamp-00.md

```diff
@@ -41,91 +41,84 @@ This document defines the Web Application Messaging Protocol (WAMP). WAMP is a r
 
 _This section is non-normative._
 
-The WebSocket protocol brought bi-directional real-time connections to the browser. This defines an API at the message level, requiring users who want to use WebSocket connections in their applications to define any semantics on top of it.
+The WebSocket protocol brings bi-directional real-time connections to the browser. It defines an API at the message level, requiring users who want to use WebSocket connections in their applications to define their own semantics on top of it.
 
 The Web Application Messaging Protocl (WAMP) was initially defined as a WebSocket sub-protocol at the technical level, and is intended to provide application developers with the semantics they need to handle messaging between components in distributed applications.
 
-WAMP is a routed protocol, with all components connecting to a WAMP router.
+WAMP is a routed protocol, with all components connecting to a _WAMP router_.
 
-WAMP provides two messaging patterns: Publish & Subscribe and routed Remote Procedure calls.
+WAMP provides two messaging patterns: _Publish & Subscribe_ and _routed Remote Procedure Calls_.
 
-Publish & Subscribe is an established messaging pattern where components inform the router that they want to receive information on a topic (they suscribe to the topic). A component can then publish to this topic, and the router distributes events to all subscribers.
+Publish & Subscribe (PubSub) is an established messaging pattern where a component, the _Subscriber_, informs the router that it wants to receive information on a topic (i.e., it subscribes to a topic). Another component, a _Publisher_, can then publish to this topic, and the router distributes events to all Subscribers.
 
```

Comment:
After restructuring the following RPC paragraph (see note for line 54), I decided to restructure the PubSub one in a consistent way.